src/sass/partials/_delivery.scss

```diff
@@ -23,11 +31,14 @@
     content: '';
     position: absolute;
     height: 1px;
-    width: calc(100% - 14px - 126px);
+    width: calc(100% - 14px - 140px);
```

Comment:
Kiedy mówiłem -140px to pytałem czemu nie -140 zamiast -14 -126px (co daje -140).
Teraz masz -154px i nie wiem czy o ten efekt chodziło :)